# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/xfree86-cvs/xterm-ls.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./programs/xterm/XTerm.ad.orig	Wed Jun 14 21:43:58 2000
+++ ./programs/xterm/XTerm.ad	Thu Dec 28 15:22:20 2000
@@ -138,3 +138,6 @@
 *VT100*color15: white
 *VT100*colorUL: yellow
 *VT100*colorBD: white
+
+! A new xTerm should (= must!) have a login shell.  (Clifford)
+*loginShell: on
